setup.ps1

```diff
@@ -0,0 +1,13 @@
+# set PSGallery as trusted so we can install packages from there
+Write-Host 'Trusting PS Gallery'
+Set-PSRepository -Name "PSGallery" -InstallationPolicy Trusted
+ 
+# Install PSAKE
+Write-Host 'Installing PSake'
+Install-Module -Name psake -Scope CurrentUser
+
+# Install dotnet based tools (requires a manifest)
+Write-Host 'Install dotnet tools'
+dotnet new tool-manifest
```

Comment:
Just do these two steps once and commit the .config folder it creates. Then next time do: `dotnet tool restore`